### DISCUSSION TITLE
How to remove space after Textarea?

### DISCUSSION THREAD

**Discussion by sshyam-gupta** [chakra-ui/chakra-ui#4087] [Q&A]
I am using FormControl + Textarea + FormHelperText but i see unnecessary space below textarea.
How to remove this?

Copy the following code snippet and paste it in any chakra-ui editable example.

```
<>
  <FormControl id="email">
    <FormLabel>Email address</FormLabel>
    <Input type="email" />
    <FormHelperText>We'll never share your email.</FormHelperText>
  </FormControl>
  
  <FormControl id="text-area-email">
    <FormLabel>Email address</FormLabel>
    <Textarea />
    <FormHelperText>We'll never share your email.</FormHelperText>
  </FormControl>
</>
```

![Screen Shot 2021-05-26 at 12 09 46 AM](https://user-images.githubusercontent.com/38554977/119551136-eb0b0e80-bdb6-11eb-9c27-d8820fe5863c.png)

**Comment by vcastroi** [comment]
FormHelperText does have a margin top you can remove by setting it up to zero, using the theme or manually on the component
weird thing is that even after removing it on the TextArea-group there is still a gap (7px) that I couldn't find where it comes from
FormLabel also has a bottom margin
both are removed here, note that I had to set the bottom FormHelperText marginTop to -7px to hide that rare gap:
```
<>
  <FormControl id="email">
    <FormLabel mb='0px'>Email address</FormLabel>
    <Input type="emailc" />
    <FormHelperText mt='0px'>We'll never share your email.</FormHelperText>
  </FormControl>
  
  <FormControl id="text-area-email">
    <FormLabel mb='0px'>Email address</FormLabel>
    <Textarea />
    <FormHelperText mt='-7px'>We'll never share your email.</FormHelperText>
  </FormControl>
</>
```
interested on what's causing that gap between the TextArea and the FormHelperText, let us know if you find out

**Reply by sshyam-gupta** [reply]
Thanks @vcastroi but I am using `FormHelperText` with other form inputs as well. 
The problem is I am unable to even inspect that space in chrome inspector.

I will inspect more and try to fix it in the proper way.

**Reply by vcastroi** [reply]
well FormHelperText  top margin can be removed by changing the theme

And just found out the problem with TextArea, seems like is caused by the browser display default for the component
can be fixed by setting the textarea style to 
```
display: block
``` 
(browser default is _display:inline-block_) 
**vertical-align: top** also seems to fix the problem
that can also be set on the theme

funny GitHub also uses that display style override on this very same textarea

hope that helps, and we learnt something today

**Reply by with-heart** [reply]
@vcastroi Want to submit a PR for that?

**Reply by sshyam-gupta** [reply]
Thanks @vcastroi . This worked! Can you please move it to the comment so I can mark it as `Answered`?

**Reply by vcastroi** [reply]
@sshyam-gupta will do
@with-heart yeah I'll try to create my first PR

**Comment by vcastroi** [comment]
And just found out the problem with TextArea, seems like is caused by the browser display default for the component
can be fixed by setting the textarea style to

display: block
(browser default is display:inline-block)
vertical-align: top also seems to fix the problem
that can also be set on the theme

funny GitHub also uses that display style override on this very same textarea

hope that helps, and we learnt something today
